Fix "driver reporting unsupported charge behaviour"
spam on the console, which also blocks the boot process.

diff --git a/drivers/power/supply/rk818_charger.c b/drivers/power/supply/rk818_charger.c
index ba673996bc4c..a17eeb34c315 100644
--- a/drivers/power/supply/rk818_charger.c
+++ b/drivers/power/supply/rk818_charger.c
@@ -619,6 +619,7 @@ static const struct power_supply_desc rk818_charger_desc = {
 	.property_is_writeable	= rk818_charger_prop_writeable,
 	.get_property		= rk818_charger_get_property,
 	.set_property		= rk818_charger_set_property,
+	.charge_behaviours = POWER_SUPPLY_CHARGE_BEHAVIOUR_AUTO | POWER_SUPPLY_CHARGE_BEHAVIOUR_INHIBIT_CHARGE,
 };
 
 // }}}
